lib/chusaku.rb

```diff
@@ -80,14 +80,31 @@ def clean_group(group)
     def annotate_group(group:, route_data:)
       whitespace = /^(\s*).*$/.match(group[:body])[1]
       route_data.reverse_each do |datum|
-        name = datum[:name]
-        annotation = "@route #{datum[:verb]} #{datum[:path]}"
-        annotation += " (#{name})" unless name.nil?
-        comment = "#{whitespace}# #{annotation}\n"
+        comment = "#{whitespace}# #{annotate_route(**datum)}\n"
         group[:body] = comment + group[:body]
       end
     end
 
+    # Generate route annotation
+    #
+    # @param {String} verb
+    # @param {String} path
+    # @param {String} name
+    # @param {Hash} defaults
+    # @return {String}
+    def annotate_route(verb:, path:, name:, defaults:, **)
```

Comment:
Love the destructured hash approach in passing the arguments to this method. I think it'll be okay to omit the trailing `**` since you've listed out all the possible arguments that will be passed from line 83. What do you think?

Reply:
We can, but then the method might break in next version (if more things would be stored in a route metadata). It will be caught by tests, though... So we can, yes.

Reply:
Done